cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,75 @@
 /// <reference types='cypress' />
 
-describe('Student Registration page', () => {
+describe( 'Student Registration page', () => {
```

Comment:
Remove the extra space before 'Student Registration page' and use single quotes instead of double quotes for consistency.